sdk/storage/storage-file-share/src/ShareFileClient.ts

```diff
@@ -248,6 +272,10 @@ export interface FileUploadRangeFromURLOptions extends CommonOptions {
    * Additional parameters for the operation
    */
   sourceConditions?: SourceModifiedAccessConditions;
+  /**
+   * Additional parameters for the operation
```

Comment:
Description is not accurate. Better like "Additional lease access conditions."

Reply:
Missing @type and @memberof tags in comment.

Reply:
Fixed.